interactive/commands.py

```diff
@@ -0,0 +1,65 @@
+from jobserver.authorization import InteractiveReporter
+from jobserver.github import _get_github_api
+from jobserver.models import OrgMembership, ProjectMembership, Repo, User
+
+
+def create_repo(*, name, get_github_api=_get_github_api):
```

Comment:
I think this is a nice way of moving logic out of the view and model, and has resulted in some cleaner code.

My only, minor, comment is that I'm always a bit wary of naming something after the particular pattern that it implements. There's usually something that expresses the intent better within that specific domain. I'm not sure what this is right now though... we're creating jobserver things, but for Interactive. Come to think of it, all of the imports are from jobserver and this code is called from jobserver too. Is it in the right place?

Things to think about, but no reason to not merge it.

Reply:
I see your point about the code all coming from job-server, and I did think about putting this module into the `jobserver` app, but they're so specific to interactive that they would all need renaming to denote that.

My long term hope/goal is that we can move all onboarding to a flow similar to this, at which point I'd move them into the main app because they're not special-cased, but until then they're specifically for an interactive flow.